vm-setup/roles/firewall/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+use_firewalld: true
+baremetal_interface: baremetal
+provisioning_interface: provisioning
+vbmc_port_range: "6230:6235"
```

Comment:
I can fix it in a follow-up if you prefer, but as previously mentioned we should avoid hard-coding this, instead we can calculate the range based on the number of masters/workers, then generate a list of port numbers to iterate over, similar to https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/common/tasks/main.yml#L6

Reply:
I replied - https://github.com/metal3-io/metal3-dev-env/pull/91#discussion_r335995099

Reply:
Sorry I missed that, my point was that we're not overriding it with any calculated value, so for NUM_WORKERS of more than 3 the ports won't be opened.

I was thinking we could just pass the number of masters/workers in to ansible and calculate the port, but I could also do it in bash like I've proposed for dev-scripts.

Not a blocker, perhaps it's easier if we fix this as a follow-up since this looks ready to go otherwise and it's just maintaining the existing behaviour.

Reply:
Ah I see. I was hoping we could do it separately, just didn't want to change too much in one PR.

Reply:
Sure that's fine, I can take care of it after this lands.